eks/outputs.tf

```diff
@@ -0,0 +1,10 @@
+#
+# Outputs
+#
+output "path-to-kubeconfig" {
+  value = "${var.kubeconfig_path}"
+}
+
```

Comment:
This is user specified, so why is this an output as well?

Reply:
I have kept it intentionally just to make user aware that the `Kubeconfig` has been saved to the specified path. We can change it if you think its redundant.